driver_cpl/driver/prep_ocn_mod.F90

```diff
@@ -988,7 +992,7 @@ subroutine prep_ocn_merge( flux_epbalfact, a2x_o, i2x_o, r2x_o, w2x_o, g2x_o, xa
                                               g2x_o%rAttr(index_g2x_Fogg_rofi , n)) * flux_epbalfact
 
 
-       if ( index_x2o_Foxx_rofl_16O /= 0 ) then
+       if ( lwiso_runoff ) then
```

Comment:
I know you have basically made lwiso_runoff a synonym for index_x2o_Foxx_rofl_16O, but this code would feel better (more symmetrical) to me if you either:

(a) kept this line as it was before

or

(b) determined the value of lwiso_runoff based on index_x2o_Foxx_rofl_16O (as you currently do), then made the use of _all_ isotope fields dependent on lwiso_runoff